R/centralization.R

```diff
@@ -288,7 +288,7 @@ centr_degree_tmax <- function(graph = NULL,
     lifecycle::deprecate_warn(
       when = "2.0.0",
       what = "centr_degree_tmax(loops = 'must be explicit')",
-      details = "Default value (`FALSE`) will be dropped in next release, add an explicit value for the loops argument."
+      details = "The default value (currently `FALSE`) will be dropped in the next release, add an explicit value for the loops argument."
```

Comment:
```suggestion
      details = "The default value (currently `FALSE`) will be dropped in the next release. Add an explicit value for the loops argument."
```

Maybe split the sentence?